fairscale/optim/oss.py

```diff
@@ -46,7 +46,9 @@ class OSS(Optimizer):
             torch.distributed group (default: group.WORLD)
     """
 
+    #: The optimizer used for a given shard
```

Comment:
just curious, is this : and extra space below significant for doc purpose?

Reply:
ah no, I think that was a reformat from the IDE, it changes depending on the black version, it's pretty annoying